virtualization-and-containers/material/exercise_vm_containers_text.md

```diff
@@ -184,6 +184,22 @@ We want to provision (`config.vm.provision`) the box in several steps. You can r
 - [Vagrant Homepage](https://www.vagrantup.com/)
 - [Vagrant Introduction](https://www.vagrantup.com/intro)
 - [VirtualBox Manual](https://www.virtualbox.org/manual/UserManual.html)
+- If you are using Vagrant on Windows you might run into the following issue/error message:
+
+  ```powershell
+  VBoxManage.exe: error: Details: code E_FAIL (0x80004005), component ConsoleWrap, interface IConsole
```

Comment:
I think there are three back ticks missing here to close this code block. I cannot add these as a suggestion because GitHub does not display these backticks.